src/utils/__tests__/githubApi.test.ts

```diff
@@ -87,101 +80,3 @@ describe('getFile', () => {
     }
   });
 });
-
-describe('codeMatches', () => {
-  test('accepts numerical code', () => {
-    expect(codeMatches(100, [100])).toBe(true);
-  });
-
-  test('accepts text code', () => {
-    expect(codeMatches(101, [HTTP_RESPONSE_1XX])).toBe(true);
-  });
-
-  test('allows single value instead of a list', () => {
-    expect(codeMatches(102, HTTP_RESPONSE_1XX)).toBe(true);
-  });
-
-  test('does not accept invalid code', () => {
-    expect(codeMatches(100, [200, HTTP_RESPONSE_2XX])).toBe(false);
-  });
-});
-
-describe('retryHttp', () => {
-  const params: Partial<RetryParams> = { cooldown: 1 };
-  const errorCode = (c: number) => ({
-    status: c,
-  });
-
-  // these are standing in for an async function (the type is () =>
-  // Promise<T>)
-  const funcReturns = async () => Promise.resolve('result');
-  const funcThrows = async () => Promise.reject(errorCode(400));
-
-  test('resolves without an error', async () => {
-    await expect(retryHttp(funcReturns, params)).resolves.toBe('result');
-  });
-
-  test('resolves after one retry', async () => {
-    const f = jest
-      .fn()
-      .mockImplementationOnce(funcThrows)
-      .mockImplementationOnce(funcReturns);
-
-    expect(
-      await retryHttp(f, { ...params, retryCodes: [400], retries: 1 })
-    ).toBe('result');
-  });
-
-  test('throws an error after max retries', async () => {
-    expect.assertions(1);
-    const f = jest
-      .fn()
-      .mockImplementationOnce(funcThrows)
-      .mockImplementationOnce(funcThrows)
-      .mockImplementationOnce(funcReturns);
-
-    try {
-      await retryHttp(f, { ...params, retryCodes: [400], retries: 1 });
-      throw Error('unreachable');
-    } catch (e) {
-      return expect(e).toEqual(errorCode(400));
-    }
-  });
-
-  test('calls the cleanup function after each retry', async () => {
-    const f = jest
-      .fn()
-      .mockImplementationOnce(funcThrows)
-      .mockImplementationOnce(funcThrows)
-      .mockImplementationOnce(funcReturns);
-    let cleanupCalled = 0;
-
-    expect(
-      await retryHttp(f, {
-        ...params,
-        cleanupFn: async () => {
-          cleanupCalled += 1;
-          return Promise.resolve();
-        },
-        retries: 2,
-        retryCodes: [400],
-      })
-    ).toBe('result');
-    expect(cleanupCalled).toBe(2);
-  });
-
-  test('throws an error if error code is not in list', async () => {
-    expect.assertions(1);
-    const f = jest
-      .fn()
-      .mockImplementationOnce(funcThrows)
-      .mockImplementationOnce(funcReturns);
-
-    try {
-      await retryHttp(f, { ...params, retryCodes: [500] });
-      throw Error('unreachable');
-    } catch (e) {
-      return expect(e).toEqual(errorCode(400));
-    }
-  });
-});
```

Comment:
This is all loosely related dead code removal.

Reply:
Yup.